README.md

```diff
@@ -103,6 +103,18 @@ Initialize the Datadog tracer when set to `true`. Defaults to `false`.
 
 Set to `true` to merge the X-Ray trace and the Datadog trace, when using both the X-Ray and Datadog tracing. Defaults to `false`.
 
+### DD_INFERRED_SPANS (experimental)
```

Comment:
Just want to confirm that if you plan to immediately release this layer after merging. If not, I would break out the docs into another PR so they can be released later.

Reply:
Yes, the plan is to release this layer immediately. I've updated the docs with your suggestions, thanks!